README.md

```diff
@@ -29,6 +29,128 @@ For available metrics please refer to the latest core's master branch
 To display pipeline diagrams make sure that `:links` and `:inits_and_terminates` telemetry flags are present,
 for certain metrics to be present on charts make sure that the desired metrics are added under `:metrics` key. 
 
+## The simplest way to set all of this up: docker-compose
+
+This requires minimal setup:
+1. Clone [this repository](https://github.com/membraneframework/membrane_telemetry_dashboard)
+2. Run `docker-compose up` inside the cloned folder
+3. Perform [this step](#wiring-a-membrane-pipeline-to-a-membrane_timescaledb_reporter)
+4. Access the dashboard at `localhost:8000`
+
+## The simplest way to set all of this up without docker-compose
+
+For this application to work, you need a couple things set up:
+
+  - a TimescaleDB instance running **on port 5432**. If the application the pipeline belongs to also wants a service running on 5432, then you need to change that.
+  - a Membrane pipeline with `membrane_timescaledb_reporter` set up to persist reports to the Timescale instance,
+  - this app
+
+### Setting up the Timescale instance
+
+This is a set of sample parameters, which should _just work_
+
+```bash
+docker run \
+-e POSTGRES_DB=membrane_timescaledb_reporter \
+-e POSTGRES_USER=postgres \
+-e POSTGRES_PASSWORD=postgres \
+--name dashboard-timescale \
+-d -p 5432:5432 \
+timescale/timescaledb:latest-pg14
+```
+
+### Wiring a membrane pipeline to a `membrane_timescaledb_reporter`
+
+This is the most complicated part of all of this. First, add `membrane_timescaledb_reporter` to your `mix.exs`:
```

Comment:
I would add an explanation that these changes need to be done in your application directory, where the membrane pipeline is run